Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 gem 'rspec'
 gem 'sinatra'
+gem 'haml'
+gem 'shotgun'
```

Comment:
I think we should put `shotgun` in development group in our Gemfile. It is not meant to be used on production. While we're at it, probably we can move `rspec` to development and test group as well